import java.util.Scanner;

public class PowerOfNum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter number ");
        int number = scanner.nextInt();

        System.out.print("Enter power: ");
        int power = scanner.nextInt();

        if (number > 0 && power > 0) {
            int result = 1;
            for (int i = 1; i <= power; i++) {
                result *= number;
            }
            
            System.out.println(result);
        } 

        scanner.close();
    }
}
